### PR TITLE
Add more information and fixes

### PR DESCRIPTION
**Pull request by anh0516**
I added some more possible fixes as well as information about the status of current issues.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +35,7 @@ Note: Some of these may be subjective, and some may have been fixed without my k
 - [ ] [OBS requires a restart (or something) for some things to work correctly, but doesn't say so.](https://youtu.be/3E8IGy6I9Wo?t=224)
 - [ ] [The option to show Snap, Flatpak, and AUR packages in Pamac is "hidden".](https://youtu.be/3E8IGy6I9Wo?t=540)
 - [ ] [Scrolling with the mouse wheel in KDE's volume mixer applet scrolls through both devices and levels of devices.](https://youtu.be/3E8IGy6I9Wo?t=573)
+  - A fix for this has already been merged.
```
Comment (neheb) [inline README.md:38]: Reference to commit would be nice

Comment (glibg10b) [reply README.md:38]: I'll merge this once someone finds a link

**Comment by magnus-ISU** [comment]
Shouldn't fixed issues have their checkboxes checked (Referring to "Scrolling with the mouse wheel in KDE's volume mixer applet scrolls through both devices and levels of change")

**Comment by glibg10b** [comment]
@magnus-ISU It's checked, just not in this pull request

File: README.md (hunk 1 above)
Comment (anh0516) [reply README.md:38]: Niccolò Venerandi said something here: https://www.youtube.com/watch?v=KBRTHsukUmY&t=633s

It's my fault for not rewatching it to recall exactly what he said. He actually said that he's not sure if a fix is merged or if someone is currently working on it. Either way, this is a known issue by the KDE devs.

**Comment by glibg10b** [comment]
The relevant items have links now

**Pull request closed without merging**